tests/check_simple_example_int.sh

```diff
@@ -58,8 +58,15 @@ else
     head -n 1 "$2" | grep -q -E "# fuzion.debugLevel=1( .*)$" && export OPT=-Dfuzion.debugLevel=1
     head -n 1 "$2" | grep -q -E "# fuzion.debugLevel=0( .*)$" && export OPT=-Dfuzion.debugLevel=0
 
+    # limit cpu time for executing test
+    cpu_time_limit=$(ulimit -t)
+    ulimit -S -t 300
+
     EXIT_CODE=$(FUZION_DISABLE_ANSI_ESCAPES=true FUZION_JAVA_OPTIONS="${FUZION_JAVA_OPTIONS="-Xss${FUZION_JAVA_STACK_SIZE=5m}"} ${OPT:-}" $1 -XmaxErrors=-1 -interpreter "$2" >tmp_out.txt 2>tmp_err.txt; echo $?)
 
+    # restore original cpu time and stack size limits
+    ulimit -S -t $cpu_time_limit
+
```

Comment:
```suggestion
    # restore original cpu time and stack size limits
    ulimit -S -t $cpu_time_limit

```

I believe there is no need to restore, since these limits apply only to the current script.